.gitignore

```diff
@@ -1,3 +1,4 @@
-_build
+/_build/
+/_opam/
```

Comment:
This probably conflicts with the change from the other PR. I'd probably suggest doing this in a separate commit if it is needed.

Reply:
Yes, the first commit is the one from #66. I decided to base it on top of that, to avoid creating a branch that will immediately generate conflicts when the other branch is merged so I'd rebase it on top of master if we decide to merge #66 in some way.

Reply:
Generally it's best to have such PRs be independently commitable if possible.